engine/class_modules/monk/sc_monk.cpp

```diff
@@ -171,6 +173,15 @@ void monk_action_t<Base>::apply_buff_effects()
   parse_effects( p()->buff.press_the_advantage );
 
   // Conduit of the Celestials
+  parse_effects( p()->buff.flight_of_the_red_crane );
+  parse_effects( p()->buff.heart_of_the_jade_serpent_cdr );
+  parse_effects( p()->buff.heart_of_the_jade_serpent_cdr_celestial );
```

Comment:
This is just a dummy spell that toggles effects on `heart_of_the_jade_serpent_cdr`.